src/components/Tooltip/Tooltip.js

```diff
@@ -58,6 +63,7 @@ export default class Tooltip extends Component {
       showIcon,
       iconName,
       iconDescription,
+      menuOffset,
```

Comment:
`direction="right"` default positioning is off. I think the default value of `menuOffset.left` may be putting the floatingMenu too far over.